jina/proto/jina.proto

```diff
@@ -317,6 +317,7 @@ message RequestProto {
             TERMINATE = 0; // shutdown the BasePod
             STATUS = 1; // check the status of the BasePod
             IDLE = 3; // used in ROUTER-DEALER pattern, tells the router that the dealer is idle
+            BUSY = 4; // used in ROUTER-DEALER pattern, tells the router that the dealer is busy (or closed)
```

Comment:
For what is described in the slides, I would call CLOSED, or CANCEL

Reply:
I thought since the request is doing the opposite of IDLE it should be called BUSY and can be used in other cases as well in the future when the dealer does not want to receive data requests for whatever reason.
But since I don't have a concrete example, I would be also fine with naming it differently.
I can rename it to CANCEL.

Reply:
What do you think?

Reply:
Okay, I changed it to CANCEL